tests/restore.sh

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+
+usage()
+{
+    echo "Usage: $0 [dbg^|rel] [x86^|amd64^|arm] [-?]"
+    exit -8
+}
+
+
+source $(cd "$(dirname "$0")"; pwd -P)/testenv.sh
+
+if [ -z ${__CoreRT_BuildOS} ]; then
```

Comment:
Why are we prefixing all CoreRT global env variables with `__` on Unix, but not on Windows?